app/experimenter/nimbus-ui/src/components/FormOverview/index.tsx

```diff
@@ -31,6 +31,12 @@ const FormOverview = ({
     mode: "onTouched",
   });
   const { isSubmitted, isValid, isDirty, touched } = formState;
+  const shouldWarnOnExit = useExitWarning();
```

Comment:
Hmm, I'm not sure this work as expected. I can make an edit on the overview page and click on "Branches" to navigate away after I've made an edit without an alert showing.

Reply:
That's fair, this appears to be an issue with how it works with React Router (maybe it doesn't send the `beforeunload` event). It works reliably with true browser navigation for me, though. We'll probably want to break this out into a bug issue.

Reply:
WFM to do in a follow up. We can ask Ana how imperative it is that they can't navigate through the sidebar, we may want to add it to the "create/edit flow" epic and if what you've got is mostly sufficient we can file an issue in the "UX nice-to-haves".

Reply:
Issue filed: https://jira.mozilla.com/browse/EXP-545